Azure/azure-cli
# [ACR] `az acr build` and `az acr task`: Highlight the difference between arguments in the description

**Related command**
`az acr build` and `az acr task`

**Description**
#28269

**Testing Guide**
`az acr build --help` and `az acr task create --help` should output the updated description.

**History Notes**


[Component Name 1] `az acr build` and `az acr task`: Highlight the difference between arguments in the description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
